theme/less/_navigation.less

```diff
@@ -1,25 +1,103 @@
-nav
+// The main navigation menu
+
+#main-navigation 
 {
-  ul
+  background: linear-gradient(#fcfcfc, #ececec);
+}
+
+#main-navigation > ul
+{
+  height:50px;
+  display: flex;
+  padding: 0;
+  border-color: #EAEAEA;
+  border-radius: 5px;
+  border-style: solid;
+  border-width: 1px;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.2);
+}
+
+#main-navigation > ul li
+{
+  padding: 15px;
+  position: relative;
+  list-style: none;
+  border-right: 1px solid #e3e3e3;    
+}
+
+#main-navigation > ul ul
+{
+  top: 50px;
+  display: block;
+  position: absolute;
+  left: 0;
+  padding: 0px;
+  background-color: #f0f0f0;
+}
+
+#main-navigation > ul ul ul
+{
+  left: 99%;
+  top: 0;
+  margin-left: 1px;
+}
+
+#main-navigation > ul li:not(:hover) > ul
+{
+  display:none;
+}
+
+#main-navigation ul ul li
+{
+  padding:10px;
+  min-width: 150px;
+}
+
+#main-navigation ul ul li:hover, ul ul a:hover
+{
+  background-color: #ffffff;
+}
+
+#main-navigation ul ul a
+{
+  text-decoration: none;
+  color: #595959;
+}
+
+#main-navigation > ul > li > a
+{
+  text-decoration: none;
+  font-weight: bold;
+  font-family: "PT Sans", "HelveticaNeue", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  color:#2883DE;
+  text-shadow: 1px 1px 0px #fff;
+}
+
+#main-navigation ul li img
```

Comment:
Suggestion: we can also use `::after` to simplify the markup and supposedly we shouldn't need to use `<img>` tag here as it doesn't provide any accessibility feature here.

Reply:
Also not sure if `float: right` is required here.

Reply:
Also, there are 2 states in the https://chrome-automation.com/ there is a hover as well.

Reply:
We can not select which menu entry has submenu. Thus we need at least non-empty `span` to add background image and select the `span for other properties.

Reply:
As I have mentioned in the comment, you can use a classname instead.
Maybe something like [in layout code](https://github.com/browser-automation/cba-website/pull/7/files#diff-18f055c29e5211ab0d148460aa1716e9R13):
```
<li <%= sub_item ? "class="has-children" : "" %>
```

And use 
`#main-navigation ul li:sub_item::after` or `#main-navigation ul li:sub_item a::after`

As an example.

Reply:
I will use here anchor class. Because `li` is part of recurrent flow and moving it or adding another `li` will cause recurrent function work not correct I think.

Reply:
Not sure if I understand your argument. Can you please elaborate on 
`Because li is part of recurrent flow and moving it or adding additional li will cause recurrent function to work not correct I think ` ?

What I can see you have just duplicated `<a>` tags in your markup:
1. This will eventually provide you much of headache when you decide to make your menu accessible, because both anchor elements are focusable, so you might need to make one not focusable which eventually will require you to create a hack to highlight whole menu item.
2. It's not necessary to duplicate those anchors, not sure how is this different/better from using `<span>` as you have done previously.

Let's talk about this issue, to understand what bother you about using solution suggested above and try to understand all pros and cons.